src/Plugin/Validation/Constraint/TimeframeConstraint.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\wmmeta\Plugin\Validation\Constraint;
+
+use Drupal\Core\Entity\Plugin\Validation\Constraint\CompositeConstraintBase;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Constraint(
+ *     id = "timeframe_constraint",
+ *     label = @Translation("Timeframe", context = "Validation"),
+ *     type = "entity:meta"
+ * )
+ */
+class TimeframeConstraint extends CompositeConstraintBase
+{
+    public $invalidTimeframe = 'The publish and unpublish dates do not form a valid timeframe';
```

Comment:
Can't we reword this to `The unpublish date has to be after the publish date`.

"do not form a valid timeframe" might confuse editors

Reply:
Oke, this has been changed